absl/status/statusor.cc

```diff
@@ -50,7 +51,7 @@ BadStatusOrAccess& BadStatusOrAccess::operator=(BadStatusOrAccess&& other) {
 }
 
 BadStatusOrAccess::BadStatusOrAccess(BadStatusOrAccess&& other)
-    : status_(std::move(other.status_)) {}
+    : status_(std::move(other.status_)) noexcept {}
```

Comment:
This doesn't build because the syntax here is wrong. Please be sure to run the tests first.